src/Umbraco.Cms.Integrations.DAM.Aprimo/Services/IAprimoService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Umbraco.Cms.Integrations.DAM.Aprimo.Models;
+
+namespace Umbraco.Cms.Integrations.DAM.Aprimo.Services
+{
+    public interface IAprimoService
+    {
+        Task<AprimoResponse<SearchedRecordsPaged>> SearchRecords(string page);
+
+        Task<AprimoResponse<Record>> SearchRecord(Guid id);
```

Comment:
This could perhaps be better named as `GetRecordById`?  Just as it's not really a search.

Reply:
Updated